honeybee_radiance/cli/sunpath.py

```diff
@@ -27,44 +26,55 @@ def sunpath():
 
 
 @sunpath.command('location')
-@click.option('--lat', default=0, type=float, show_default=True,
+@click.option(
+    '--lat', default=0, type=float, show_default=True,
     help='The latitude of the location in degrees. Values must be between -90 and 90.'
     ' Default is set to the equator.')
-@click.option('--lon', default=0, type=float, show_default=True,
+@click.option(
+    '--lon', default=0, type=float, show_default=True,
     help='The longitude of the location in degrees')
-@click.option('--tz', default=0, type=float, show_default=True,
+@click.option(
+    '--tz', default=0, type=float, show_default=True,
     help='A number representing the time zone of the location you are constructing. This'
     ' can improve the accuracy of the resulting sun plot.  The time zone should follow'
     ' the epw convention and should be between -12 and +12, where 0 is at Greenwich, UK,'
     ' positive values are to the East of Greenwich and negative values are to the West.'
     )
-@click.option('--north', default=0, type=float, show_default=True,
+@click.option(
+    '--north', default=0, type=float, show_default=True,
     help='Angle to north (0-360). 90 is west and 270 is east'
     )
-@click.option('--start-date', default='JAN-01', show_default=True,
+@click.option(
+    '--start-date', default='JAN-01', show_default=True,
```

Comment:
Is it worth considering merging start-date and start-time into one input variable `start-date-time`? Dito for end-date.

Reply:
That can give a wrong impression to people. JAN-01-8:00 to JAN-15-20:00 will give an impression that every hour between Jan 1st at 8am to Jan 15 and 8pm will be included. However the time that will be included is the hours between 8am-8pm between Jan 1 to Jan 15. For instance Jan1 at 10 pm is not included.

Reply:
Right, that makes sense!